libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -196,6 +206,29 @@ void CodeTransform::operator()(assembly::Literal const& _literal)
 	checkStackHeight(&_literal);
 }
 
+CodeTransform::CodeTransform(
+	AbstractAssembly& _assembly,
+	AsmAnalysisInfo& _analysisInfo,
+	bool _julia,
+	bool _evm15,
+	ExternalIdentifierAccess const& _identifierAccess,
+	int _stackAdjustment,
+	std::shared_ptr<CodeTransform::Context> _context
+):
+	m_assembly(_assembly),
+	m_info(_analysisInfo),
+	m_julia(_julia),
+	m_evm15(_evm15),
+	m_identifierAccess(_identifierAccess),
+	m_stackAdjustment(_stackAdjustment),
+	m_context(_context)
+{
+	m_builtinFunctions["abort"] = solidity::Instruction::INVALID;
```

Comment:
Should this really be a member or rather just a static variable in the cpp file?